src/pages/edit-profile/EditProfile.tsx

```diff
@@ -213,7 +271,41 @@ const EditProfile = () => {
     if (hash) {
       navigate(`${authRoutes.myProfile.path}#complete`)
     }
-  }
+  }, [profileState, changedFields])
```

Comment:
Fix linter errors here. This function can have unexpectable behaviour because of missing dependencies
![image](https://github.com/user-attachments/assets/66a23fa2-2c21-4114-9056-0ed93820a570)